projects/canopy/src/styles/typography.scss

```diff
@@ -1,5 +1,53 @@
 @import '../styles/mixins.scss';
 $fonts-path: '../assets/fonts' !default;
+$fonts-path-lyon: '/assets/fonts' !default;
```

Comment:
Because this is a `default` does that mean that people could potentially override this property if they are including the scss files? Just wondering if we document that?

Reply:
I've removed the `default`, I think it just came from duplicating the line above to edit it. Having it there does indicate that this var could be changed, but in reality it should stay the same.